react/features/base/modal/components/JitsiKeyboardAvoidingView.js

```diff
@@ -27,6 +28,11 @@ type Props = {
      */
     hasTabNavigator: boolean,
 
+    /**
+     * Is the text input covered by android device keyboard?
+     */
+    isInputCoveredByAndroidKeyboard: boolean,
```

Comment:
Maybe something like `hasTextInput` is more descriptive.

Reply:
This is how I named it first, but it's not appropriate because AddPeople screen has an input and also Participants has an input. I don't want to cover those screens.

Reply:
Why is it not needed on those screens? just because it's not at the bottom? `hasBottomTestInput` then ?

Reply:
Yeah, this works. I will rename it this way.